src/App.php

```diff
@@ -207,20 +207,26 @@ public function readModel(Config $config): array
             $bucketParts = explode('.', $config->getBucket());
             $storageClient->createBucket(substr($bucketParts[1], 2), $bucketParts[0]);
         }
+        $mapping = [];
         foreach ($model['dataSets'] as $tableId => $d) {
             $dataFile = "{$this->temp->getTmpFolder()}/$tableId.csv";
+            $columns = array_keys($d['columns']);
             $csv = new CsvFile($dataFile);
-            $csv->writeRow(array_keys($d['columns']));
+            $csv->writeRow($columns);
             $storageClient->createTable($config->getBucket(), $tableId, $csv);
             $configuration['tables']["{$config->getBucket()}.$tableId"] = $d;
+            $mapping[] = ['columns' => $columns, 'source' => "{$config->getBucket()}.$tableId"];
```

Comment:
ak premenna `$tableId ` skutocne obsahuje tableId tak by sa uz nemala concatnut s `$config->getBucket()` nie? alebo je v tableId meno tabulky?

Reply:
jj. je to bez bucketu, měl bych tu proměnnou spíš přejmenovat.

Reply:
on je to vlastně gooddatí identifier: https://github.com/keboola/gooddata-writer-v3/blob/ead5dbed03d198d106bdd65d134e3c2534ffab9c/src/ModelReader.php#L116-L117, datová tabulka ve Storagi se pak pojmenuje podle toho.

Reply:
Proměnnou jsem přejmenoval na `$tableName` (v kontextu toho kódu v `App.php`).